packages/pwa-kit-react-sdk/src/ssr/universal/components/link/index.jsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Link as RouterLink} from 'react-router-dom'
+import {matchRoute} from '../../utils'
+import routes from '../../routes'
+import {getAppOrigin} from '../../../../utils/url'
+
+/**
+ *  The Link component match the URL with the paths defined in routes.jsx
+ *  if the URL doesn't match returns an anchor element. if it match returns a
+ *  react-router component with a relative URL.
+ *
+ */
+const Link = (props) => {
+    const {to, toHostname = getAppOrigin(), ...rest} = props
+    let _routes = routes
+    if (typeof routes === 'function') {
+        _routes = routes()
+    }
+
+    //TODO: Extend Link to use {Link as RouterLink} OR {NavLink as RouterNavLink} from react-router
+
+    // Remove routes with * path
+    // TODO: Use a RegExp
+    _routes = _routes.filter((_route) => _route.path !== '*')
+
+    const isMatch = matchRoute(to, _routes).match
+    return isMatch ? <RouterLink to={to} {...rest} /> : <a href={`${toHostname}${to}`} {...rest} />
```

Comment:
I'm wondering if we can simplify the component by removing the `toHostname` prop. Is it possible to make `to` to accept both relative and absolute URL? If `to` is absolute URL, then by default, we assume it's external link and use the native anchor tag `a`.

it could be something like: 
```
const isMatch = matchRoute(to, _routes).match
const isAbsolute = isAbsolute(to)
return isMatch && !isAbsolute  ? <RouterLink to={to} {...rest} /> : <a href={`${to}`} {...rest} />
```

Reply:
Summary of our chat. I removed the `toHostname` is not required and adds complexity.

The original problem that we want to solve is to reduce the manual steps to implementing Hybrid in the PWA.
If developers have to manually update all the Links URLs through the PWA App to use absolute URLs, we’re not reducing the manual work.
We reduce the number of manual steps by using the `<Link />` component from the SDK. Then the only manual step is to remove routes from the PWA `routes.jsx` file. All link URLs not matching the PWA routes will point to eCDN.